org.gnome.Totem.json

```diff
@@ -219,6 +219,7 @@
                 "-Dsystemd_user_services=false",
                 "-Dwriteback=false",
                 "-Dbattery_detection=none",
+                "-Dlandlock=disabled",
```

Comment:
This can be squashed into the tracker update commit as well, please keep the reference in the commit though.